configcatclient/configcatclient.py

```diff
@@ -171,10 +195,13 @@ def set_default_user(self, user):
     def clear_default_user(self):
         self._default_user = None
 
-    def stop(self):
+    def close_resources(self):
         if self._cache_policy:
             self._cache_policy.stop()
-        ConfigCatClient.sdk_keys.remove(self._sdk_key)
+
+    def close(self):
+        self.close_resources()
+        ConfigCatClient._instances.pop(self._sdk_key)
```

Comment:
Should we consider to prepare for [this](https://github.com/configcat/.net-sdk/pull/50) case?